### PR TITLE
Terraform configuration for alerts and synthetics

### PR DESCRIPTION
**Pull request by zstix**
## Description
We need to adjust which account owns the browser agent, sytnethetics monitor, and related alert conditions. Unfortunately, entities and policies can not be moved between accounts - they need to be recreated. I'd like to take another shot at configuring this stuff with _Terraform_ and automating the application of it using [atlantis](https://www.runatlantis.io/).

## Reviewer Notes
This is not a high-priority item, but it would be nice to get it set up sometime in the next two weeks.

### CODE REVIEW

File: terraform/main.tf [hunk 1]
```diff
@@ -0,0 +1,61 @@
+terraform {
+  required_version = "~> 0.13.0"
+
+  required_providers {
+    newrelic = {
+      source  = "newrelic/newrelic"
+      version = "~> 2.12"
```
Comment (jpvajda) [inline terraform/main.tf:7]:
```suggestion
      version = "~> 2.21"
```

Comment (jpvajda) [reply terraform/main.tf]: bumped this to the most current TF version so we can leverage dashboard apis... 🥳 https://github.com/newrelic/terraform-provider-newrelic/releases/tag/v2.21.0

**Review by jpvajda — COMMENTED** [review]
just suggested a bump in TF provider version

**Comment by zstix** [comment]
We've set this up manually for now. I'm going to close the PR, but keep the branch open.

**Pull request closed without merging**